applications/service_brokers/installing-ansible-service-broker.adoc

```diff
@@ -0,0 +1,22 @@
+[id='sb-installing-asb']
+= Installing the {asb-name}
+include::modules/common-attributes.adoc[]
+:context: sb-installing-asb
+
+toc::[]
+
+You can install the {asb-name} to gain access to the service bundles that it
+provides.
+
+IMPORTANT: The {asb-name} is deprecated in this release. Equivalent and better functionality is present in the Operator Framework and
+Operator Life Cycle Manager (OLM).
```

Comment:
Samely concern with @jboyd01 , hope confirm that we are using exact word.
@shawn-hurley